scripts/actions/mobskills/absorbing_kiss.lua

```diff
@@ -10,6 +10,15 @@ mobskillObject.onMobSkillCheck = function(target, mob, skill)
 end
 
 mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    if mob:getPool() == 3116 then -- BCNM20 Charming Trio: Pepper does all attribute drain
+        for i = 136, 142 do
+            xi.mobskills.mobDrainAttribute(mob, target, i, 8, 3, 120) -- Can drain up to -8 per stat
```

Comment:
Even with the comment this is very hard to parse. This is handled a little better in other places:
```lua
    -- str down - chr down
    local effectType = math.random(136, 142)

    skill:setMsg(xi.mobskills.mobDrainAttribute(mob, target, effectType, 10, 3, 120))
```

But it could be even easier:

```
enum/effects.lua:
    STR_DOWN                 = 136,
    DEX_DOWN                 = 137,
    VIT_DOWN                 = 138,
    AGI_DOWN                 = 139,
    INT_DOWN                 = 140,
    MND_DOWN                 = 141,
    CHR_DOWN                 = 142,

for effect = xi.effect.STR_DOWN, xi.effect.CHR_DOWN do
   ...
```

Reply:
Updated for clarity